src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsConfiguration.cs

```diff
@@ -43,6 +43,8 @@ public NotificationHubsConfiguration()
         /// </summary>
         public string HubName { get; set; }
 
+        internal TraceWriter Trace { get; set; }
```

Comment:
Can you either (a) make it private, or (b) just pass the context.Trace directly to your function like I do here: https://github.com/Azure/azure-webjobs-sdk-extensions/blob/master/src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs#L55. It cleans up the class a little bit.